### DISCUSSION TITLE
Is MQTT required for TeslaMate basic functionality?

### DISCUSSION THREAD

**Discussion by ylexus** [teslamate-org/teslamate#2250] [Q&A] [answered]
I am running TeslaMate from Docker. I am not planning to subscribe to data via mqtt. If I disable the MQTT service by commenting it out in `docker-compose.yaml` (and restarting), will it break any basic functionality?
Reason is that this is all running on my RPi and I already have a mqtt broker running on the same port, plus it’s a waste of resources.

**Comment by DrMichael** [comment, marked as answer]
You should probably set `DISABLE_MQTT=true` in the teslamate section...

**Reply by ylexus** [reply]
Looks to be working, thanks.
